### DISCUSSION TITLE
4kn-sector-disk-passthrough-in-proxmox/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#233] [Announcements]
# 4kn-sector-disk-passthrough-in-proxmox/

Solving an issue with 4Kn sector disks and passthrough in Proxmox with direct KVM arg injection.

https://blog.ktz.me/4kn-sector-disk-passthrough-in-proxmox/

**Comment by jjross** [comment]
Forgive me but how would you know whether the disk is 4Kn?
